salt/modules/boto_dynamodb.py

```diff
@@ -252,6 +252,25 @@ def update_global_secondary_index(table_name, global_indexes, region=None,
     return table.update_global_secondary_index(global_indexes)
 
 
+def list_tables(region=None, key=None, keyid=None, profile=None):
+    '''
+    List all DynamoDB tables visible to the supplied credentials.
+
+    CLI example::
```

Comment:
It's helpful for our doc output to use:
```
CLI Example:

.. code-block:: bash
```
Instead of `CLIE example::`

Would you mind changing that?